app/javascript/components/artists/ArtistProfile.jsx

```diff
@@ -68,6 +67,27 @@ class ArtistProfile extends React.Component {
     window.location = `/works/${work_id}/edit`;
   }
 
+  updateFeatured = (work_id) => {
+    let formData = new FormData();
+    const formKeys = ['featured_work_id'];
+    formKeys.forEach(key => {
```

Comment:
nit: since `formKeys` is only length 1, the `forEach` loop is probably not necessary